packages/app/src/components/profile/UserChallengesList.tsx

```diff
@@ -224,75 +224,121 @@ export const UserChallengesList = (props: Props): React.ReactElement => {
+                            onPress={returnToProfile}
+                        >
+                            <Icon
+                                name="chevron-left"
+                                size={42}
+                                color={colors.pink}
+                                style={styles.icon}
+                            />
+                        </TouchableOpacity>
+                    )}
+                </View>
 
-            <View>
-                <Title>Challenges</Title>
+                <View style={styles.background}>
+                    <Title style={styles.title}>Challenges</Title>
 
-                <SectionList
-                    sections={sortChallengesData()}
-                    renderItem={renderItem}
-                    renderSectionHeader={({ section: { title } }) => (
-                        <Text>{title}</Text>
-                    )}
-                    // keyExtractor={(item) => item.node._id}
-                    onEndReached={loadNext}
-                    onRefresh={refetchList}
-                    refreshing={isFetchingTop}
-                    ItemSeparatorComponent={() => <View style={null} />}
-                    ListFooterComponent={null}
-                />
+                    <SectionList
+                        sections={sortChallengesData()}
+                        renderItem={renderItem}
+                        renderSectionHeader={({ section: { title } }) => (
+                            <Text style={styles.sectionTitle}>{title}</Text>
+                        )}
+                        // keyExtractor={(item) => item.node._id}
+                        onEndReached={loadNext}
+                        onRefresh={refetchList}
+                        refreshing={isFetchingTop}
+                        ItemSeparatorComponent={() => <View style={null} />}
+                        ListFooterComponent={null}
+                    />
+                </View>
             </View>
-        </View>
+        </Gradient>
     );
 };
 
 const styles = StyleSheet.create({
+    title: {
+        textAlign: 'center',
+        fontSize: 34,
+        margin: '5% 0%'
+    },
+    sectionTitle: {
+        fontSize: 17,
+        color: 'white',
+        fontWeight: '500',
+        marginBottom: '4%'
+    },
+    cardWrapper: {
+        position: 'relative',
+        borderColor: 'white',
+        borderWidth: 3,
+        borderRadius: 5,
+        borderStyle: 'solid',
+        marginBottom: 20
+    },
     container: {
-        flex: 1,
-        backgroundColor: '#e6ffff',
-        width: '100%',
-        height: '100%'
+        paddingHorizontal: '4%',
+        flex: 1
+    },
+    background: {
+        paddingHorizontal: '5%',
+        marginTop: '15%',
+        borderColor: 'rgba(251, 250, 250, 0.7)',
+        borderWidth: 1,
+        borderTopLeftRadius: 10,
+        borderTopRightRadius: 10,
+        backgroundColor: 'rgba(255, 255, 255, 0.5)'
```

Comment:
Let's add `flex: 1` here to pull the background all the way done even without any list items. I saw that I missed this in the market list as well and will be updating it. 

![Screen Shot 2021-08-10 at 12 11 03 PM](https://user-images.githubusercontent.com/19338130/128916135-0a3fff81-927c-4135-bf1c-ad29139bf395.png)